bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java

```diff
@@ -404,34 +401,28 @@ private String updateKonnectedModule() throws KonnectedHttpRetryExceeded {
     /**
      * Sends a command to the module via {@link KonnectedHTTPUtils}
      *
-     * @param scommand the string command, either 0 or 1 to send to the actutor pin on the Konnected module
+     * @param command the state to send to the actuator
      * @param zone the zone to send the command to on the Konnected Module
      */
-    private void sendActuatorCommand(Integer scommand, String zone, ChannelUID channelId) {
+    private void sendActuatorCommand(OnOffType command, String zone, ChannelUID channelId) {
         try {
             Channel channel = getThing().getChannel(channelId.getId());
             if (channel != null) {
                 logger.debug("getasstring: {} getID: {} getGroupId: {} toString:{}", channelId.getAsString(),
                         channelId.getId(), channelId.getGroupId(), channelId);
                 ZoneConfiguration zoneConfig = channel.getConfiguration().as(ZoneConfiguration.class);
                 KonnectedModuleGson payload = new KonnectedModuleGson();
-                payload.setState(scommand);
-
                 payload.setZone(thingID, zone);
 
-                // check to see if this is an On Command type, if so add the momentary, pause, times to the payload if
-                // they exist on the configuration.
-                if (scommand == zoneConfig.onValue) {
+                if (command == OnOffType.ON) {
+                    payload.setState(zoneConfig.onValue);
                     payload.setTimes(zoneConfig.times);
-                    logger.debug("The times configuration was set to: {} for channelID: {}.", zoneConfig.times,
```

Comment:
The serialized payload is logged below, and provides more accurate information than the individual debug lines.